pom.xml

```diff
@@ -437,6 +438,16 @@
                 <artifactId>netty-leak-detector-junit-extension</artifactId>
                 <version>${netty-leak-detector-junit-extension.version}</version>
             </dependency>
+            <dependency>
```

Comment:
Can we discuss (perhaps on Kroylicious slack) what the merits of the framework are?  Currently it seems like it is provided a couple of enums and some environment variable utility class.